src/components/footer/Footer.js

```diff
@@ -1,7 +1,105 @@
-import React from "react";
+import React from 'react';
+import Nav from '../nav/Nav';
+import './Footer.scss';
 
 function Footer() {
-  return <div></div>;
+  return (
+    <div className="footer-Wrap">
+      <div className="footer-Aside">
+        <img
+          src={process.env.PUBLIC_URL + '/images/Footer/footerImg1.png'}
```

Comment:
어떤 부분을 환경변수로 활용하신 걸까요??
img 태그의 src 속성으로 들어가는 경로는 public 폴더 하위의 절대경로이기 때문에 아래처럼만 써주셔도 됩니다!
```suggestion
          src='/images/Footer/footerImg1.png'
```